### DISCUSSION TITLE
WebUI: "Add to album" - recent is not recent

### DISCUSSION THREAD

**Discussion by SwallowYourDreams** [immich-app/immich#11051] [Feature Request]
### The bug

Whenever a user adds a photo to an album, Immich will show

1. the three most "recent" albums at the top of the list
1. a list of all albums, ordered from most to least recently-created.

My issue is with the "recent" category. There are several ways how you could consider an album "recent":

1. Most recently *created*
2. Most recently *modified* (album renamed, photos added, removed)

To me, it seems like Immich only shows the most recently-created albums at the top of the list. This, however, isn't particularly useful, as the list of all albums below it will also show the most recently-created albums at the top of the list. Hence, the category "recent" is perfectly redundant. Instead, I would expect it to follow definition no. 2 and show the albums that were most recently *modified*.

I'm not quite sure if this is truly a bug or works as intended. If it's intended, it's certainly bugging *me* because it's redundant and I keep scrolling past the useless "recent" category to get to what I want to see. For the mods: feel free to convert this into a feature request if it turns out it's not a bug.

### The OS that Immich Server is running on

Linux Mint 21.3

### Version of Immich Server

1.107.2

### Version of Immich Mobile App

1.107.1

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Irrelevant
```

### Your .env content

```Shell
Irrelevant
```

### Reproduction steps

```bash
1. Go to photos view.
2. Select any photo.
3. Select the "+" sign menu option in the top right corner.
4. Select "Add to album".
5. Add to any album *not* recently created. Let's call it "2021", created in 2021.
6. Repeat the process with a different image until and including step 4; you will not see the album "2021" in the list, even though it has just been modified.
```

### Relevant log output

```shell
None
```

### Additional information

None

**Comment by VenomousViper27** [comment]
I'm seeing this issue as well, adding pictures to a "Picture Frame" album and this album isn't showing in the "recents" category as I would expect (and make it easier to choose being at the top of the list).
